flagd/cmd/start.go

```diff
@@ -18,16 +18,17 @@ import (
 const (
 	corsFlagName           = "cors-origin"
 	logFormatFlagName      = "log-format"
-	metricsExporter        = "metrics-exporter"
 	managementPortFlagName = "management-port"
+	metricsExporter        = "metrics-exporter"
 	otelCollectorURI       = "otel-collector-uri"
 	portFlagName           = "port"
 	serverCertPathFlagName = "server-cert-path"
 	serverKeyPathFlagName  = "server-key-path"
 	socketPathFlagName     = "socket-path"
 	sourcesFlagName        = "sources"
+	syncEnabledFlagName    = "syncEnabled"
```

Comment:
should we change this to `sync-enabled` and `sync-port`, to stay in line with the other flags which are all in kebap-case?

Reply:
Good point, will change to kebab-case

Reply:
done this the commit https://github.com/open-feature/flagd/pull/1237/commits/d7c5798048864dcc504fffafbdebbf61fbdf1938